planner/cascades/transformation_rules_test.go

```diff
@@ -157,3 +157,24 @@ func (s *testTransformationRuleSuite) TestTopNRules(c *C) {
 	s.testData.GetTestCases(c, &input, &output)
 	testGroupToString(input, output, s, c)
 }
+
+func (s *testTransformationRuleSuite) TestProjectionElimination(c *C) {
+	s.optimizer.ResetTransformationRules(map[memo.Operand][]Transformation{
+		memo.OperandProjection: {
+			NewRuleEliminateProjection(),
+		},
+		memo.OperandDataSource: {
+			NewRuleEnumeratePaths(),
```

Comment:
I think this is unnecessary. We only need `NewRuleEliminateProjection` here?

Reply:
But it is ok to be here. Don't need to change it then.

Reply:
OK, I just copied from the upper test function. Fixed. The test output simplified a lot.